src/components/course/course-header/CourseHeader.jsx

```diff
@@ -50,6 +51,24 @@ const CourseHeader = () => {
   );
   const hasSufficientReviewCount = courseReviews?.reviewsCount >= 5;
 
+  if (hasSufficientReviewCount && isExperimentVariationA) {
+    sendEnterpriseTrackEvent(
+      enterpriseConfig.uuid,
+      'edx.ui.enterprise.learner_portal.course.viewedWithReviewsVariation',
+      {
+        course_key: course.key,
+      },
+    );
+  } else {
+    sendEnterpriseTrackEvent(
+      enterpriseConfig.uuid,
+      'edx.ui.enterprise.learner_portal.course.viewedWithoutReviewsVariation',
+      {
+        course_key: course.key,
+      },
+    );
+  }
```

Comment:
[clarification] I believe these `sendEnterpriseTrackEvent` calls run the risk of being called on every component re-render that matches the conditional rather than just when the dependent values change.

Wondering if this `sendEnterpriseTrackEvent` should be moved into a `useEffect`?